disnake/interactions/modal.py

```diff
@@ -85,15 +85,23 @@ def __init__(self, *, data: InteractionPayload, state: ConnectionState):
         super().__init__(data=data, state=state)
         self.data = ModalInteractionData(data=data["data"])  # type: ignore
 
+    def walk_components(self) -> Iterable[NestedComponent]:
```

Comment:
doesn't this return a generator?

Reply:
It returns an iterator which is pretty much an iterable

Reply:
Nvm it does